source/containers.tex

```diff
@@ -4155,7 +4155,7 @@
 \begin{itemdescr}
 \pnum
 \effects If \tcode{sz < distance(begin(), end())}, erases the last \tcode{distance(begin(),
-end()) - sz} elements from the list. Otherwise, inserts \tcode{sz - distance(begin(), end())}  
```

Comment:
Could you please submit the trailing whitespace deletions as a separate pull request?